### PR TITLE
Saved Searches Page

### PR DESCRIPTION
**Pull request by mjoyce91**
Restyles the Saved Searches page using existing components from the Saved Searches addition to the Profile Dashboard.

Once this is good to merge, we can also include it in the Sprint 17 branch.

### CODE REVIEW

File: src/Components/SavedSearches/SavedSearchesList/SavedSearchesList.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SavedSearchesListResultsCard from '../SavedSearchesListResultsCard';
+// import Alert from '../../Alert';
```
Comment (burgwyn) [inline src/Components/SavedSearches/SavedSearchesList/SavedSearchesList.jsx]: Remove comment

File: src/Components/SavedSearches/SavedSearchesListResultsCard/SavedSearchesListResultsCard.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const SavedSearchListResultsCard = ({ savedSearch, goToSavedSearch, mappedParams,
+deleteSearch }) => {
+  const pills = mapSavedSearchToDescriptions(savedSearch.filters, mappedParams);
+  const dateCreated = savedSearch.date_created ?
+    formatDate(savedSearch.date_created) : NO_LAST_UPDATED_DATE;
+  return (
+    <div className="usa-grid-full saved-search-card profile-section-container" key={savedSearch.id}>
+      <div className="usa-grid-full">
+        <div className="usa-width-one-fourth">
+          <div><span className="title">Name: </span><span>{savedSearch.name}</span></div>
```
Comment (burgwyn) [inline src/Components/SavedSearches/SavedSearchesListResultsCard/SavedSearchesListResultsCard.jsx]: Let's `h3` the name

File: src/Containers/SavedSearchesMap/SavedSearchesMap.test.jsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ import SavedSearchesList from '../../Components/ProfileDashboard/SavedSearches/S
 const middlewares = [thunk];
 const mockStore = configureStore(middlewares);
 
-describe('SavedSearchesContainer', () => {
+describe('SavedSearchesMapContainer', () => {
```
Comment (burgwyn) [inline src/Containers/SavedSearchesMap/SavedSearchesMap.test.jsx]: 👍

**Comment by burgwyn** [comment]
This is a free text search with "German"
screen shot 2018-03-19 at 10 25 16 pm

**Comment by mjoyce91** [comment]
@burgwyn - good point on the free text not being a pill. I could include that as one of the pills, since a lot of people will probably just search by keyword.

**Comment by burgwyn** [comment]
I'm assuming it is available in the API response.  I think representing it 💊"German"/💊 is fine.

**Comment by mjoyce91** [comment]
Right, yeah it is. Initially I followed the logic of the Results Page pills, which obviously don't show the search term as a pill. But I can see how it makes sense to render it here.

**Comment by mjoyce91** [comment]
There was an issue where the the requests made by the Search header and the Saved Searches were stepping over each other and pills were getting deleted, so I made a fix for that as well.

**Pull request merged**